example/configuration.nix

```diff
@@ -0,0 +1,63 @@
+{ config, pkgs, ... }:
+let
+  unstableTarball =
+    fetchTarball
+      https://github.com/NixOS/nixpkgs-channels/archive/nixos-unstable.tar.gz;
+in
+{
+  imports = [ <nixpkgs/nixos/modules/installer/virtualbox-demo.nix> ];
+
+  nix = {
+    binaryCaches = [
+      "https://cache.nixos.org/"
+      "https://all-hies.cachix.org"
+    ];
+    binaryCachePublicKeys = [
+      "all-hies.cachix.org-1:JjrzAOEUsD9ZMt8fdFbzo3jNAyEWlPAwdVuHw4RD43k="
+    ];
+  }; 
+
+  nixpkgs.config = {
+    packageOverrides = pkgs: {
+      unstable = import unstableTarball {
+        config = config.nixpkgs.config;
+      };
+    };
+  };
+
+  nixpkgs.config.allowUnfree = true;
+
+  # Let demo build as a trusted user.
+# nix.trustedUsers = [ "demo" ];
+
+# Mount a VirtualBox shared folder.
+# This is configurable in the VirtualBox menu at
+# Machine / Settings / Shared Folders.
+# fileSystems."/mnt" = {
+#   fsType = "vboxsf";
+#   device = "nameofdevicetomount";
+#   options = [ "rw" ];
+# };
+
+# By default, the NixOS VirtualBox demo image includes SDDM and Plasma.
+# If you prefer another desktop manager or display manager, you may want
+# to disable the default.
+# services.xserver.desktopManager.plasma5.enable = lib.mkForce false;
+# services.xserver.displayManager.sddm.enable = lib.mkForce false;
+
+# Enable GDM/GNOME by uncommenting above two lines and two lines below.
+# services.xserver.displayManager.gdm.enable = true;
+# services.xserver.desktopManager.gnome3.enable = true;
+
+# Set your time zone.
+# time.timeZone = "Europe/Amsterdam";
+
+# List packages installed in system profile. To search, run:
+# \$ nix search wget
```

Comment:
Maybe delete the commented out example stuff in here? Not sure how useful it is for someone who isn't really configuring their own system.

Reply:
yeah, couldn't decide, will delete